mockgcp/mockcompute/regionalbackendservicev1.go

```diff
@@ -41,7 +41,7 @@ func (s *RegionalBackendServicesV1) Get(ctx context.Context, req *pb.GetRegionBa
 
 	obj := &pb.BackendService{}
 	if err := s.storage.Get(ctx, fqn, obj); err != nil {
-		return nil, err
+		return nil, status.Errorf(codes.NotFound, "The resource '%s' was not found", fqn)
```

Comment:
I don't think we want to capitalize these -> `the resource ...`

Reply:
I made this change to make sure the mock message matches the realGCP.